Library/Homebrew/dependency.rb

```diff
@@ -168,6 +168,16 @@ def merge_temporality(deps)
         [] # Means both build and runtime dependency.
       end
     end
+
+    def expand_dependency(dependency)
```

Comment:
Yeah if you need to pass it down to the `expand` call, then you have to accept it in the signature as well.